.ui/library/flDialog.cpx

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
we switched to native alert iirc. Could you remove this?

Reply:
done.